package.json

```diff
@@ -1,31 +1,40 @@
 {
-  "name": "probot-civicrm",
+  "name": "civicrmdev-bot",
```

Comment:
You'll have to rebase this file onto Tim's original and then maybe only add only new packages.

Reply:
Oh Yes, I updated this file completely by mistake. Thanks for reviewing and letting me know about the same. I will update this right away.